### PR TITLE
DS-288 Add the custom icons fonts

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

File: src/components/icons/IconWrapper.tsx
Comment (gregaubert) [inline src/components/icons/IconWrapper.tsx:1]: Not sure about the pattern here for supporting two different font-families, if you have better ideas feel free to suggest

File: src/components/icons/IconWrapper.tsx [hunk 1]
```diff
@@ -47,3 +47,21 @@ const StyledSpan = styled.span`
...
+  font-family: 'Echoes Icons';
+  font-size: 1.2em;
+  font-weight: normal;
+  font-style: normal;
+  line-height: 1;
+  letter-spacing: normal;
+  text-transform: none;
+  white-space: nowrap;
+  word-wrap: normal;
+  direction: ltr;
```
Comment (david-cho-lerat-sonarsource) [inline src/components/icons/IconWrapper.tsx:62]: do we really need this one (`direction`) ?

Comment (gregaubert) [reply src/components/icons/IconWrapper.tsx:62]: Probably not, this whole style comes from the class provided by the material UI lib, there are probably many things we can clean up.

**Review by david-cho-lerat-sonarsource — APPROVED** [review]
Cool. Maybe we can also display IconCloud in Storybook?

File: src/components/icons/IconWrapper.tsx
Comment (jeremy-davis-sonarsource) [reply src/components/icons/IconWrapper.tsx:1]: I don't see any better pattern

**Comment by gregaubert** [comment]
> Cool. Maybe we can also display IconCloud in Storybook?

We can add once we merge with the Story from Jay, I don't have it yet on this PR

**Review by jeremy-davis-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**